app/src/main/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -30,6 +30,15 @@ class AuthRepository @Inject constructor(
 		firebaseAuth.signOut()
 	}
 
+	fun photoUrl() =
```

Comment:
Shouldn't we have one method and one data class that fetch all the user data at once ? I could imagine that we will have 20 values connected with the user, we will probably want to avoid 20 functions:D especially when all the data here is some sort of a user profile data :)

Reply:
Yeah, I have the same feeling, we should fetch it all at once and return some profile model 😄

Reply:
Done 😄